case/src/main/kotlin/com/ritense/case/web/rest/CaseTabManagementResource.kt

```diff
@@ -30,12 +32,12 @@ import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 
 @RequestMapping("/api/management", produces = [APPLICATION_JSON_UTF8_VALUE])
-class CaseTabManagementResource(
+open class CaseTabManagementResource(
     private val caseTabService: CaseTabService
 ) {
 
     @PostMapping("/v1/case-definition/{caseDefinitionName}/tab")
-    fun createCaseTab(
+    open fun createCaseTab(
```

Comment:
functions don't need to be specified `open` when the class is already `open`, right?

Reply:
Unfortunately, if even a single function here is not `open`, I get an error.

All the `open` in this class are needed to make sure the one `@RunWithoutAuthorization` works.